source/client/process_impl.cc

```diff
@@ -46,6 +47,25 @@ using namespace std::chrono_literals;
 namespace Nighthawk {
 namespace Client {
 
+// We customize ProdClusterManagerFactory for the sole purpose of returning our specialized
+// http1 pool to the benchmark client, which allows us to offer connection prefetching.
+class ClusterManagerFactory : public Envoy::Upstream::ProdClusterManagerFactory {
+public:
+  using Envoy::Upstream::ProdClusterManagerFactory::ProdClusterManagerFactory;
+
+  Envoy::Http::ConnectionPool::InstancePtr
+  allocateConnPool(Envoy::Event::Dispatcher& dispatcher, Envoy::Upstream::HostConstSharedPtr host,
+                   Envoy::Upstream::ResourcePriority priority, Envoy::Http::Protocol protocol,
+                   const Envoy::Network::ConnectionSocket::OptionsSharedPtr& options) override {
+    if (protocol == Envoy::Http::Protocol::Http11 || protocol == Envoy::Http::Protocol::Http11) {
```

Comment:
Note to self: one of these ought to test for http1.0

Reply:
Yes :)